packages/realm-server/migrations/1728561353454_create-user-billing.js

```diff
@@ -0,0 +1,128 @@
+exports.up = (pgm) => {
+  pgm.createTable('users', {
+    id: 'id',
```

Comment:
I wonder if this should be a UUID? (this is shorthand for a sequence I believe) every time I've not implemented user ID's as UUID's i've regretted it.

Reply:
Actually that's a good point, I'll change it to a UUID.

Reply:
I changed all ids to uuids.